optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-client/src/test/java/org/optaplanner/workbench/screens/solver/client/editor/SolverEditorPresenterTest.java

```diff
@@ -47,10 +50,17 @@
 import org.uberfire.backend.vfs.ObservablePath;
 import org.uberfire.backend.vfs.Path;
 import org.uberfire.ext.editor.commons.client.history.VersionRecordManager;
+import org.uberfire.ext.editor.commons.service.support.SupportsSaveAndRename;
+import org.uberfire.mvp.Command;
 import org.uberfire.mvp.PlaceRequest;
 import org.uberfire.workbench.events.NotificationEvent;
 
-import static org.mockito.Mockito.*;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
 @RunWith(GwtMockitoTestRunner.class)
 public class SolverEditorPresenterTest {
```

Comment:
I know not due your changes, but there is unused import:
- import javax.enterprise.util.TypeLiteral;

Would you mind to remove?

Reply:
No worries, will we reformat the complete repository and turn on checkstyle soon.

Reply:
Good! I removed the `import javax.enterprise.util.TypeLiteral` anyway ;-)